src/components/cluster/detail/AddNodePool.js

```diff
@@ -0,0 +1,488 @@
+import { connect } from 'react-redux';
+import { nodePoolCreate } from 'actions/nodePoolActions';
+import AWSInstanceTypeSelector from '../new/aws_instance_type_selector';
+import Button from 'UI/button';
+import NodeCountSelector from 'shared/node_count_selector';
+import NumberPicker from 'UI/number_picker';
+import produce from 'immer';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import styled from '@emotion/styled';
+
+const WrapperDiv = styled.div`
+  background-color: ${props => props.theme.colors.shade7};
+  padding: 20px;
+  border-radius: 5px;
+  margin-bottom: 40px;
+  h3 {
+    margin-bottom: 20px;
+  }
+`;
+
+const FlexWrapperDiv = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin-bottom: 13px;
+  & > div:nth-of-type(2) > button {
+    padding-top: 9px;
+    padding-bottom: 9px;
+  }
+  button {
+    margin-right: 16px;
+  }
+  .availability-zones & {
+    margin-bottom: 22px;
+  }
+  p {
+    margin-left: 15px;
+  }
+`;
+
+const FlexColumnDiv = styled.div`
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  margin: 0 auto;
+  max-width: 650px;
+  label {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
+    margin: 0 0 31px;
+    &.instance-type {
+      margin-bottom: 21px;
+    }
+    p {
+      line-height: 1.4;
+    }
+  }
+  .label-span {
+    color: ${props => props.theme.colors.white1};
+  }
+  .label-span,
+  input,
+  select {
+    font-size: 16px;
+    margin-bottom: 13px;
+    font-weight: 400;
+  }
+  input {
+    box-sizing: border-box;
+    width: 100%;
+    background-color: ${props => props.theme.colors.shade5};
+    padding: 11px 10px;
+    outline: 0;
+    color: ${props => props.theme.colors.whiteInput};
+    border-radius: 4px;
+    border: 1px solid ${props => props.theme.colors.shade6};
+    padding-left: 15px;
+    line-height: normal;
+  }
+  & > input {
+    /* margin-bottom: 13px !important; */
+  }
+  p {
+    margin: 0;
+    font-size: 14px;
+    color: ${props => props.theme.colors.white1};
+  }
+  a {
+    text-decoration: underline;
+  }
+  /* Overrides for AWSInstanceTypeSelector */
+  .textfield label,
+  .textfield,
+  .textfield input {
+    margin: 0;
+  }
+  /* Overrides for NumberPicker */
+  .availability-zones {
+    & > div > div,
+    & > div > div > div {
+      margin: 0;
+    }
+    & div > p {
+      font-size: 16px;
+    }
+  }
+  .scaling-range {
+    form {
+      label {
+        margin-bottom: 7px;
+        color: ${props => props.theme.colors.white1};
+        font-weight: 400;
+      }
+      & > div:nth-of-type(2) {
+        display: none;
+      }
+    }
+  }
+`;
+
```

Comment:
This component is basically copy/pasted from `CreateCluster` component and adapted